src/Widgets/VimeoVideoWidget/VimeoVideoWidgetStyles.scss

```diff
@@ -0,0 +1,7 @@
+.vimeo-fullsize-iframe {
```

Comment:
FYI in https://medium.com/seek-blog/block-element-modifying-your-javascript-components-d7f99fcab52b I see a naming convention that uses the component name for CSS namespacing.

* [ ] We should think about a similarly strict naming convention, to prevent misuse of class names.

`VimeoVideoWidgetComponent__fullsize-ifame`, `VimeoVideoWidget__fullsize-ifame`, or so.